CONTRIBUTING.md

```diff
@@ -1,54 +1,89 @@
-# Updating Action Dockerfile
+# Steps to release
 
-If any of the shell scripts are altered in a change to the GitHub Action, the Dockerfile which they are contained in must be rebuilt, and references to the Dockerfile must be updated.
+## Step 1. Log into docker ghcr.io
 
-## Pushing image to GitHub Container Registry
-
-To push a new copy of the Action Dockerfile to GitHub Container Registry, you will first need to [authenticate to GHCR with a PAT](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
```

Comment:
linking the instructions here before was helpful

Reply:
added